### PR TITLE
Feature/codesnippet

### PR DESCRIPTION
**Pull request by MekalaNagarajan-Centrica**
- [x] hide code snippet top border radius.
- [x] show copy button.

### CODE REVIEW

File: src/components/NucleusCodeSnippet.js [hunk 1]
```diff
@@ -41,12 +44,24 @@ export class NucleusCodeSnippet extends LitElement {
...
           ${baseStyles ?? ''}
+          ${`
+            .expressive-code .frame {
+              border-radius: ${borderRadiusStyles}
+            }
+            .expressive-code pre {
+              border-radius: ${borderRadiusStyles}
+            }
+            .expressive-code .copy button {
+              opacity: initial;
```
Comment (andij) [inline src/components/NucleusCodeSnippet.js:55]: Here is some CSS trickery to add the content from the `title` attribute to display 'Copy to clipboard'.

```suggestion
              opacity: initial;
              position: relative;
              border: 1px solid rgb(225 228 232 / .4);
              
              &::before {
                pointer-events: auto;
                border: none;
                content: attr(title);
                opacity: 1;
                color: rgb(225 228 232);
                width: max-content;
                top: 50%;
                transform: translate(min(calc(-100% - 0.5rem)), -50%);
              }
```

File: src/components/starlight/MarkdownContent.astro [hunk 2]
```diff
@@ -223,5 +223,9 @@ const classList = [
     .tab a {
       text-wrap: nowrap;
     }
+
+    .expressive-code .copy button {
+      opacity: initial;
+    }
```
Comment (andij) [inline src/components/starlight/MarkdownContent.astro:229]: Is this CSS required? The same is present in NucleusCodeSnippet.js [line 55].

Comment (MekalaNagarajan-Centrica) [reply src/components/starlight/MarkdownContent.astro:229]: This is for any codeblock in markdown with 3 backticks .
```
```

File: src/components/NucleusCodeSnippet.js (hunk 1 above)
Comment (MekalaNagarajan-Centrica) [reply src/components/NucleusCodeSnippet.js:55]: I would like to keep the content short as 'Copy'.

**Review by andij — APPROVED** [review]

**Pull request merged**